src/locales/en_US.ts

```diff
@@ -163,7 +163,8 @@ const strings = {
   fragment_transaction_list_confirmation_progress: '%s of %s Confirmations',
   fragment_transaction_list_transaction: 'Transactions',
   transaction_list_buy_crypto_message: 'Buy %s',
-  transaction_list_no_tx_yet: 'No transactions yet!',
+  transaction_list_no_tx_yet: 'No transactions yet.',
```

Comment:
If you're going to remove the exclamation mark (and with it the implication of future excitement when the tx history list becomes populated) just get rid of it instead of replacing it with the killjoy period.